src/release.ss

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env gxi-script
+;; -*- Gerbil -*-
+;;; Release script for Gerbil.
+;; Usage: ./release.ss [version]
+;;
+;; Use it from a fully committed git tree.
+;;
+;; If a release version is specified (contains no "-"), a new release commit is made and tagged.
+;; If a development version is specified, a new development commit is made and tagged.
+;;
+;; If no version is specified, one is autodetected:
+;; If the current commit is a development commit, a release commit will be made and tagged
+;; If the current commit is a release commit, a development commit will be made and tagged
+;; e.g. v0.12-DEV-666-g1234567 will be turned into v0.13
+;; and v0.13 will be turned into v0.13-DEV
+
+(import
+  :gerbil/gambit/ports
+  :std/format :std/misc/ports :std/srfi/13 :std/srfi/14 :std/sugar :std/pregexp)
+
+;; Where the root to gerbil is.
+(def gerbil-root
+  (path-normalize (path-expand ".." (path-directory (this-source-file)))))
+
+;; Change the directory to the root of gerbil.
+(current-directory gerbil-root)
+
+;; Check that .git is there indeed
+(or (file-exists? ".git") (error "Cannot find .git directory"))
```

Comment:
`(unless (file-exists? ...) (error ...))` is the more idiomatic way to write this; you are not returning a useful value anywhere.